api-tests/features/p2-f2f-journey.feature

```diff
@@ -138,3 +138,68 @@ Feature: P2 F2F journey
       # Initial journey
       When I get an 'temporarily_unavailable' OAuth error from the CRI stub
       Then I get a 'pyi-technical' page response
+
+    Scenario Outline: Async queue error
+      When I get an error from the async CRI stub
+      Then I get a 'page-face-to-face-handoff' page response
+
+      # Return journey
+      When I start a new 'medium-confidence' journey and return to a 'pyi-f2f-technical' page response
+      When I submit a 'end' event
+      Then I get an OAuth response
+
+  Scenario: f2f PYI escpae route
```

Comment:
Typo: escape

(Where possible I think we should also try and be consistent about capitalising F2F in text)